src/test/resources/features/combotests/CONTINUATION-POST.feature

```diff
@@ -0,0 +1,24 @@
+Feature: This feature tests all of the requirements for the Envoy Proxy Limiter - as combinations of requests telephone, ipAddress and uprn
+  I want to test Fulfilment Journeys using the Rate Limiter for all combinations
+
+  Scenario Outline: Combinations TEST - CONTINUATION POST HH Non-Individual
```

Comment:
Please consider adding cucumber tags to all the scenarios, like we do in RHcuc and CCcuc.

Reply:
Added